src/sbn_probability.cpp

```diff
@@ -264,24 +275,37 @@ EigenVectorXd SBNProbability::ExpectationMaximization(
       // Last line of the section on EM in doc/tex.
       score_history[em_idx] += m_tilde_for_positive_alpha.dot(sbn_parameters);
     }
+    // SHJ: Print the score in the below line.
+    // Uncommenting it but leaving it incase we need to do further debugging.
+    // Remove it at some point.
+    //std::cout << std::setprecision(20) << score_history[em_idx] << std::endl;
```

Comment:
I like to take out code comments like this but put them as comments on the GH pull request.

Reply:
Taken out.